app/lib/geo.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Shared geographic calculations
+module Geo
+  class << self
+    # calculates distance between two coordinates
+    # abstracted from Geocoder
+    def dist_btn_coords(lat1, long1, lat2, long2)
```

Comment:
I have strong preference against unnecessary abbreviation. It took me multiple reads to realize that this was supposed to be `distance_between_coordinates` and not `distance_button_coordinates`.

Since the grouping of coordinates is so important here, I would prefer either passing in two arrays or using named parameters here.

Reply:
Ideally, I could use this with any two objects that respond to `#lat` and `#long`. So the method would be like
```
def distance_between(location, other_location)
  haversine_distance_between(location.lat, location.long, other_location.lat, other_location.long)
end
```
That way I could pass in a zip_code and an agency, or two agencies or a foodbank and an agency, and the method would just work as long as the objects that are being passed in define a lat and a long method.

Reply:
Interesting idea.  Need to think about this.  It would be nice if all of the objects had the same method name.  We normalize that in the serializer but I think using the model makes more sense.  Maybe alias or something.

Reply:
Implemented in latest commit.